CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to pgcopydb
+
+We're happy you want to contribute! You can help us in different ways:
+
+* Open an [issue](https://github.com/dimitri/pgcopydb/issues) with suggestions
+  for improvements, potential bugs, etc.
+* Fork this repository and submit a pull request
+
+### Building from source code
+
+Follow the relevant docs at our
+[README.md](https://github.com/dimitri/pgcopydb/#installing-pgcopydb)
+
+### Following our coding conventions
+
+CI pipeline will automatically reject any PRs which do not follow our coding
+conventions. The easiest way to ensure your PR adheres to those conventions is
+to use the
+[citus_indent](https://github.com/citusdata/tools/tree/develop/uncrustify) tool.
+This tool uses `uncrustify` under the hood.
+
+
+```bash
+# Use your favorite package manager, and install `uncrustify` by running one of
+# the following
+apt-get install uncrustify
+brew install uncrustify
+apk add --upgrade uncrustify
+dnf install uncrustify # may require epel-release rpm
```

Comment:
Let's avoid showing the alternatives within a single bash script, (some) people will inevitably copy/paste the whole thing and wonder about it. Might even report an issue...